src/util.ts

```diff
@@ -279,21 +283,16 @@ export function getManifestsFromRegistry(): Promise<string[]> {
   });
 }
 
-function removeManifestFromRegistry(manifestPath: string): Promise<any> {
+async function removeManifestFromRegistry(manifestPath: string): Promise<any> {
   if (!manifestPath) {
     return new Promise((resolve, reject) => {
       return reject('No manifest was specified');
     });
   }
   console.log(`Removing ${manifestPath}`);
-  return new Promise(async (resolve, reject) => {
-    try {
-      const [parsedType, parsedGuid, parsedVersion] = await parseManifest(manifestPath);
-      resolve(await querySideloadingRegistry(['Remove-ItemProperty -LiteralPath $RegistryPath -Name "' + parsedGuid + '" -ErrorAction SilentlyContinue']));
-    } catch (err) {
-      reject(err);
-    }
-  });
+  const [parsedType, parsedGuid, parsedVersion] = await parseManifest(manifestPath);
+  await querySideloadingRegistry(['Remove-ItemProperty -LiteralPath $RegistryPath -Name "' + manifestPath + '" -ErrorAction SilentlyContinue']);
```

Comment:
Please add a comment that explains that it needs to remove manifest path since the old code could add it using the manifest path.